### PR TITLE
kommander: Add FederatedAddonsDeployment CRD

### PR DESCRIPTION
**Pull request by makkes**
refs https://jira.mesosphere.com/browse/DCOS-62180

### CODE REVIEW

**Comment by makkes** [comment]
> Please update the `templates/federated-crds.yaml` file to include the changes suggested here. I would like to see posted here a example deployment to a cluster showing that these crds were deployed as expected.

Just deployed the chart and everything looks fine:
```
$ helm install ./stable/kommander --set kommander-cluster-lifecycle.certificates.issuer.selfSigned=true --name kommander
NAME:   kommander
LAST DEPLOYED: Tue Jan  7 21:17:34 2020
NAMESPACE: default
STATUS: DEPLOYED

[...]
$ 
```

File: stable/kommander/templates/federated-crds.yaml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {{- if .Values.kubefed.enabled }}
 {{- if not (.Capabilities.APIVersions.Has "types.kubefed.io/v1beta1") }}
 {{.Files.Get "files/federated-addons.yaml" }}
+{{.Files.Get "files/federated-addonsdeployments.yaml" }}
```
Comment (alejandroEsc) [inline stable/kommander/templates/federated-crds.yaml]: awesome thank you!

**Comment by makkes** [comment]
> oh looks like you need to bump the version in the chart file.

Dang, someone outpaced me. Fixed.

**Comment by jimmidyson** [comment]
I'm currently refactoring the KCL chart to include federated CRDs in there rather than in this kommander chart. I'm happy for this to be merged as-is to unblock, and I will incorporate this change into the KCL chart.

**Pull request merged**